torch/csrc/jit/ir/ir.h

```diff
@@ -72,6 +72,9 @@ using namespace ::c10::attr;
 namespace aten {
 using namespace ::c10::aten;
 }
+namespace cuda {
+using namespace ::c10::cuda;
+}
```

Comment:
What is this for?

Reply:
This is for the `cuda` namespace. It doesn't work if I remove this.